<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskQueryWidget</class>
 <widget class="QWidget" name="TaskQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_allTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_oneTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_oneParcel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_taskTime">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>211</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_taskSelect">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_parcelSelect">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>所有任务信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择任务编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择包裹单号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>所有任务预计到达时刻表</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
